deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -107,6 +129,38 @@
 - type: replace
   path: /instance_groups/name=api/jobs/name=staticfile-buildpack/properties?/quarks/bpm/processes
   value: []
+{{- if .Values.features.suse_buildpacks.enabled }}  
```

Comment:
I embedded this code into the code I proposed above, so this block goes away completely.